embedded-service/src/type_c/comms.rs

```diff
@@ -11,3 +11,23 @@ pub struct DebugAccessoryMessage {
     /// Connected
     pub connected: bool,
 }
+
+/// UCSI connector change message
+#[derive(Copy, Clone, Debug)]
+#[cfg_attr(feature = "defmt", derive(defmt::Format))]
+pub struct UcsiCiMessage {
```

Comment:
Should this be `UcsiCci` (`Cci`, not `Ci`)?
```suggestion
pub struct UcsiCciMessage {
```